data/FrancescoXX.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Francesco Ciulla",
+  "url": "https://francescociulla.com",
+  "image": "https://github.com/FrancescoXX.png"
```

Comment:
This line is missing a comma.

```suggestion
  "image": "https://github.com/FrancescoXX.png",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
great spot!